apps/ws-rust/src/ auth/mod.rs

```diff
@@ -0,0 +1,23 @@
+use jsonwebtoken::{decode, Validation, DecodingKey};
+use serde_json::Value;
+
+pub async fn extract_user_id(token: String) -> String {
+    let validation = Validation::default();
+    let decoded = decode::<Value>(
```

Comment:
Decode to a proper struct, never use `Value` unless absolutely needed

Reply:
cool,  ill make change